tensorizer/serialization.py

```diff
@@ -1555,6 +1555,7 @@ def __init__(
                     for name, entry in self._metadata.items()
                     if filter_func(name)
                 }
+            num_tensors: int = len(self._metadata)
```

Comment:
Looks like an undocumented change/fix?

Reply:
That's part of a small internal optimization that limits the number of `plaid_mode` buffers to the total number of tensors. I included it because there's no reason to allocate more than that anyway, and it would prevent potentially spiking RAM use up to 8x for code working on single-tensor files (for whatever reason) due to switching to `plaid_mode=True`.